how-to-guides/neptune-docker/scripts/training.py

```diff
@@ -0,0 +1,93 @@
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torchvision import datasets, transforms
+import neptune.new as neptune
+import os
+
+
+# read docker secret as environment
+with open(os.environ['NEPTUNE_API_TOKEN']) as file:
+    neptune_token = file.read() 
```

Comment:
neptune_token => api_token
``api_token`` -> we use it in multiple places in the docs